eclass/cmake.eclass

```diff
@@ -127,7 +129,11 @@ case ${CMAKE_MAKEFILE_GENERATOR} in
 esac
 
 if [[ ${PN} != cmake ]]; then
-	BDEPEND+=" dev-util/cmake"
+	if [[ ${EAPI} == 7 ]]; then
+		BDEPEND+=" dev-util/cmake"
+	else
+		BDEPEND+=" >=dev-util/cmake-3.18"
```

Comment:
Is there any harm in raising the dep unconditionally? FWICS it's the only stable version we have now.

Reply:
If no one complains re changing deps without revision, no harm I could think of.